test/Mono.Linker.Tests.Cases/Warnings/CanDisableWarnAsError.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Warnings
+{
+	[SkipKeptItemsValidation]
+	[SetupLinkerSubstitutionFile ("CanDisableWarnAsErrorSubstitutions.xml")]
+	[SetupLinkerArgument ("--verbose")]
+	[SetupLinkerArgument ("--warnaserror")]
+	[SetupLinkerArgument ("--warnaserror-", "IL2011,IL2012,IgnoreThis")]
+	[LogContains ("error IL20(07|08|09|10)", true)]
+	[LogContains ("warning IL20(11|12)", true)]
```

Comment:
Don't we need to list these one by one? Currently this will match all of the errors/warnings, but there's no validation that it got them all. For example if for some reason the `IL2012` won't be treated as warning, but `IL2011` still is, the test would still pass.